extensions/notebook/src/jupyter/serverInstance.ts

```diff
@@ -143,16 +145,32 @@ export class PerNotebookServerInstance implements IServerInstance {
 		return this._uri;
 	}
 
+	public get attachedEditorCount(): number {
+		return this._attachedEditorCount;
+	}
+
 	public async configure(): Promise<void> {
 		await this.configureJupyter();
 	}
 
+	public incrementAttachedEditorCount(): void {
+		this._attachedEditorCount++;
+		console.log('current attached editor count: ' + this._attachedEditorCount);
```

Comment:
Debug? I'd just remove it unless you see value in having this for retail

Reply:
Removed